sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -228,6 +230,87 @@ TEST_F(KeyVaultKeyClient, CreateRsaHsmKey)
     EXPECT_FALSE(keyResponse.Value.Properties.ReleasePolicy.HasValue());
   }
 }
+std::string BinaryToHexString(std::vector<uint8_t> const& src)
+{
+  static constexpr char hexMap[]
+      = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+  std::string output(static_cast<size_t>(src.size()) * 2, ' ');
+  const uint8_t* input = src.data();
+
+  for (size_t i = 0; i < src.size(); i++)
+  {
+    output[2 * i] = hexMap[(input[i] & 0xF0) >> 4];
+    output[2 * i + 1] = hexMap[input[i] & 0x0F];
+  }
+
+  return output;
+}
+
+TEST_F(KeyVaultKeyClient, ReleaseKey)
+{
+  auto const keyName = GetTestName() + "2";
+  auto const& client = GetClientForTest(keyName);
+
+  Azure::Security::Attestation::AttestationClient attestationClient(
+      "https://sharedwus.wus.attest.azure.net");
+
+  MaaSandboxClient sandboxClient("https://maasandboxsvc.azurewebsites.net");
+  attestationClient.RetrieveResponseValidationCollateral();
+
+  auto rsaKey = CreateRsaKeyOptions("rsaKeyName");
+  rsaKey.KeySize = 2048;
+  rsaKey.ExpiresOn = std::chrono::system_clock::now() + std::chrono::hours(24 * 365);
+
+  client.CreateRsaKey(rsaKey);
+
+  KeyVaultKey cloudRsaKey = client.GetKey("rsaKeyName").Value;
+  cloudRsaKey.Properties.Id = "rsaKeyName";
+  Azure::Core::Json::_internal::json destJson;
+  Azure::Core::Json::_internal::json destJson2;
+  Azure::Security::KeyVault::Keys::_detail::JsonWebKeySerializer::JsonWebKeySerialize(
+      cloudRsaKey.Key, destJson2);
+  destJson["keys"].emplace_back(destJson2);
+  auto dest = destJson.dump();
+  std::string result = Base64Url::Base64UrlEncode(std::vector<uint8_t>(dest.begin(), dest.end()));
+  auto generatedToken = sandboxClient.GenerateQuote(AttestationDataType::Json, result);
+  attestationClient.RetrieveResponseValidationCollateral();
+
+  auto attestResponse = attestationClient.AttestOpenEnclave(
+      generatedToken.Value,
+      AttestOptions{AttestationData{
+          std::vector<uint8_t>(dest.begin(), dest.end()), AttestationDataType::Binary}});
+
+  Azure::Security::KeyVault::Keys::CreateKeyOptions options;
+  options.KeyOperations.push_back(Azure::Security::KeyVault::Keys::KeyOperation::Sign);
+  options.KeyOperations.push_back(Azure::Security::KeyVault::Keys::KeyOperation::Verify);
+  options.ReleasePolicy = KeyReleasePolicy();
+  options.ReleasePolicy.Value().Immutable = false;
+  // cspell:disable
+  std::string dataStr = "{"
+                        "\"anyOf\" : [ {"
+                        "\"allOf\" : [{\"claim\" : \"x-ms-sgx-mrsigner\", \"equals\" : \""
+      + BinaryToHexString(attestResponse.Value.Body.SgxMrSigner.Value())
+      + "\""
+        "} ],"
+        "\"authority\" : \"https://sharedwus.wus.attest.azure.net\""
```

Comment:
Since you're using the MAA instance URL in two different places, it should be a manifest constant.

Reply:
done